app/scripts/lib/transaction/eip5792.ts

```diff
@@ -530,9 +541,11 @@ async function getAlternateGasFeesCapability(
     const { isSupported = false, relaySupportedForChain } = chainBatchSupport;
 
     const isSmartTransaction = getIsSmartTransaction(chainId);
+    const isSendBundleSupported = sendBundleSupportedChains[chainId] ?? false;
 
     const alternateGasFees =
       simulationEnabled &&
+      isSendBundleSupported &&
```

Comment:
In addition to the capability, we also have to update `useIsGaslessSupported` so the UI knows whether to enable gas fee token selection.